README.md

```diff
@@ -150,15 +150,15 @@ OR
         
 5. Within onResponse function you will get response object, from response object you can extract body:
 
-  ```java response.body()```
+     ```java response.body()```
 
 6. Once you do that you will get APIResponse object this object has function getDataClass, this function will return List of specified class parsed from APIResponse, for example, you can use it like this:
 
-    ```java [APIResponse object].getDataClass(Flight.class)```
+    ```[APIResponse object].getDataClass(Flight.class)```
     
     OR
           
-    ```java [APIResponse object].getDataClass(Place.class)```
+    ```[APIResponse object].getDataClass(Place.class)```
     
           Although you need to make sure that APIResponse object has data to parse these objects from. For example, you should not try to use getDataClass(Flight.class) on APIResponse object that you received by calling getPlacesCall on retrofit. Since it will not return enough information to extract Flight object from APIResponse.
```

Comment:
I think you might need to remove some spaces in front of "Althougth" otherwise it seems to be treating it as a code block
![Screen Shot 2021-09-10 at 20 47 27](https://user-images.githubusercontent.com/56419970/132935403-08c20816-f7db-482b-b829-0f9057c22a17.png)